src/satellite_images_nso_datascience/training/train.py

```diff
@@ -5,9 +5,10 @@
 import mlflow
 import pandas as pd
 from sklearn.base import ClassifierMixin, TransformerMixin
+from sklearn.metrics import classification_report
```

Comment:
classification_report and train_test_split are not used in this file and therefore should not be added.